sources/communitysuite.json

```
@@ -0,0 +1,9 @@
+{
+  "id": "CSUITE",
```

id should match file name with underscores for spaces, if applicable, e.g.
```suggestion
  "id": "COMMUNITYSUITE",
```